pkg/nsx/services/common/init.go

```diff
@@ -2,28 +2,27 @@ package common
 
 import (
 	"github.com/vmware/vsphere-automation-sdk-go/runtime/bindings"
+	"k8s.io/client-go/tools/cache"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/vmware-tanzu/nsx-operator/pkg/config"
 	"github.com/vmware-tanzu/nsx-operator/pkg/nsx"
+	"github.com/vmware-tanzu/nsx-operator/pkg/util"
 )
 
```

Comment:
the init.go looks more like a types.go, 	is Converter *bindings.TypeConverter multithread safe? It seems not good to create a instance in a common module. each module may have its own converter

Reply:
OK, that makes sense, for safety, it's better get a new instance everytime.